src/panel/ServicePanelMobile.jsx

```diff
@@ -0,0 +1,104 @@
+/* global _ */
+import React from 'react';
+import { Fragment } from 'react';
+import Panel from 'src/components/ui/Panel';
+import CategoryService from 'src/adapters/category_service';
+import CategoryList from 'src/components/CategoryList';
+import nconf from '@qwant/nconf-getter';
+
+const directionConf = nconf.get().direction;
+
+class ServicePanelMobile extends React.Component {
+  render() {
+    return <Panel
+      resizable
+      title=""
+      minimizedTitle={_('Unfold to see quick actions', 'service panel')}
+      className="service_panel"
+      white
+    >
+      <div className="service_panel__actions">
+
+        <h3>{_('Itineraries', 'service panel')}</h3>
+
+        {
+          nconf.get().direction.enabled &&
+          <Fragment>
+            <button
+              onClick={() => { window.app.navigateTo('/routes/?mode=driving'); }}
+              className="service_panel__action service_panel__item__direction">
+              <div className="service_panel__action__icon">
+                <span className="icon-drive"/>
+              </div>
+              <div className="service_panel__action__title">{_('by car', 'service panel')}</div>
```

Comment:
We need to include translations for these new strings in this PR.

Reply:
Usually we wait for a branch to be merged to apply new translations separately (Travis will upload the new translation keys to Transifex after merging to master), although that's not ideal.

Reply:
I vote for having the translations commits happening only on Master. o/

Reply:
Ah yes, I always forget the process…
That means if we want to merge in master only code that can be deployed to production as soon as possible, we have to anticipate this. Merging some PR with new keys before?